app/models/appointment.rb

```diff
@@ -16,6 +16,9 @@ class Appointment < ApplicationRecord
 
   belongs_to :rebooked_from, class_name: Appointment
 
+  scope :cancelled, -> { where(status: [:cancelled_by_customer, :cancelled_by_pension_wise]) }
```

Comment:
Same with these scopes can we provide the `status` entries using the `%i(...)` literal notation?